src/pages/chat.tsx

```diff
@@ -1,18 +1,49 @@
+//@ts-nocheck
 import type { NextPage } from 'next';
 import Image from 'next/image';
 import avatarMale from '../images/avatar/male.png';
 import Chat from '../components/chat';
-import ChatBox from '../components/chatbox';
+import { useState, useRef } from 'react';
+import { useSession, signOut } from 'next-auth/react';
+
+// Temporarily store messages of a user
+const messages = [];
 
 const ChatPage: NextPage = (props): JSX.Element => {
+    const { data: session } = useSession();
+    const dummy = useRef();
```

Comment:
Lol can we rename this into something like chat section instead? Kevin might see this and wonder what we mean by this.

Reply:
At the very least leave a comment explaining what this is.

Reply:
I used a better name and left comments to help other developers understand in the commit I just pushed.